SwiftCharts/Axis/ChartAxisLayerDefault.swift

```diff
@@ -173,10 +227,26 @@ class ChartAxisLayerDefault: ChartAxisLayer {
         }
     }
 
+    /**
+     Finds the location for the scalar value within the bounds of the axis layer
+
+     - parameter scalar:          The axis value's scalar value
+     - parameter firstAxisScalar: The first axis value's scalar value, used to find how many "steps" away the given scalar value is from the first value
+
+     - returns: The location of the axis value within the bounds of the axis layer
+     */
     func innerScreenLocForScalar(scalar: Double, firstAxisScalar: Double) -> CGFloat {
         return self.length * CGFloat(scalar - firstAxisScalar) / self.modelLength
     }
-    
+
+    /**
+     Calculates the location for the scalar value in the chart's coordinates.
+
+     - parameter scalar:          An axis value's scalar value
+     - parameter firstAxisScalar: The first axis value's scalar value, used to find how many "steps" away the given scalar value is from the first value
+
+     - returns: The location along the axis' dimension that the axis value should be displayed at
+     */
```

Comment:
It should be: - returns: The screen location where the axis value should be displayed at